edina/src/data/Situ/json/biography.json

```diff
@@ -0,0 +1,1232 @@
+            "Michael has received numerous prestigious national awards including major award: \"Moulton Museum Award\"- Laguna BeachPlein Air Paintings Invitational 2023. \"Silver Medel Award\" Master Signature Division - Oil Painters of America",
+            "Eastern Regional Juried Exhibition 2023. \"Best Oil Award\" - Carmel Art Festival-Plein Air Juried Painting Competition 2023. \"Award of Excellence\" for Master Signature Member - Oil Painters of America National Juried Exhibition 2022. \"Director\u2019s Choice Award\" and \"Kinsman Family Foundation Award\" and \"Best Quick Draw Award\" - Laguna Beach Plein Air Paintings Invitational 2021, 2020 and 2019. \"Artist Choice Award\" and \"Second Place Award\" - Carmel Art Festival 2019 and 2009. \"Seascape Award of Excellence\" - Oil Painters of America National Juried Exhibition 2018, 2014. \"Best Quick Draw Award\" - Carmel Art Festival 2018. \"Second Place Award\" - Annual Final Plein Air Salon Competition 2018. \"First Place Award\" - Carmel Art Festival 2017 and 2012. \"Award of Excellence\" - American Impressionist Society National Juried Exhibition 2012. \"Best Paintings Award\" - La Quinta Art Festival 2011. \"People\u2019 Choice Award\" - San Luis Obispo Plein Air Festival 2010. \"First Place Quick Draw Award\" - San Clemente Plein Air Painting Competition 2010. \"Award of Excellence\" - Oil Painters of America National Juried Exhibition 2008 and 2006. \"First Place Award\" - Huntington Beach Art Center Plein Air Festival 2007. \"First Place Award\" - San Clemente Plein Air Painting Competition 2007. \"Artist Choice Award\" - Sonoma Plein Air Painting Exhibition 2004. \"Museum Purchase Award\" - Maritime Museum Plein Air Paintings Exhibition 2002. \"Best of Show Award\" - La Quinta Desert Plein Air Competition 2001 and many more.",
+            "He represented by Michael Situ Studio Gallery in Laguna Beach California."
+        ]
+    },
+    {
+        "id":2,
+        "title":"Awards",
+        "content":[
+            {
+                "time":"Oct, 2023",
+                "name":"\"Moulton Museum Award\"",
```

Comment:
Remove `\"` in name

Reply:
I want to change them to ',because sometime "" is useful, like:
"name": "\"Shirl Smithson Founders' Award of Excellence\" Master Signature Division",

Reply:
Are you using JSON.stringify or similar function to generate json output?